templates-pipelines/windows-bios-installer/configmaps/windows-bios-installer-configmaps.yaml

```diff
@@ -0,0 +1,116 @@
+---
```

Comment:
Sometimes we have the YAML doc start, sometimes not? Might make sense to run `yamllint` over this repo?